modal/cls.py

```diff
@@ -212,6 +212,41 @@ def _new_user_cls_instance(self):
         user_cls_instance._modal_functions = instance_methods
         return user_cls_instance
 
+    async def update_autoscaler(
+        self,
+        min_containers: Optional[int] = None,
+        max_containers: Optional[int] = None,
+        scaledown_window: Optional[int] = None,
+        buffer_containers: Optional[int] = None,
+    ) -> None:
+        """Override the current autoscaler behavior for this Cls instance.
```

Comment:
Should we start calling a "Cls instance" a Service soon? So much easier to pronounce

Reply:
Maybe? I still don't think I understand what the plan is for changes to the client object model.